tests/test_pagination_translators.py

```
@@ -0,0 +1,828 @@
+            # First call with limit=20 fails with 400
+            if call_count == 1 and parameters.get("limit") == "20":
+                raise RequestException(status_code=400, message="Invalid limit")
+
+            # Retry with default limit succeeds
+            mock_response = AsyncMock()
+            mock_response.status = 200
+            mock_response.text = AsyncMock(return_value='{"items": [' + ",".join(str(i) for i in range(10)) + "]}")
+            return mock_response
+
+        # Mock rnet.Client properly with side_effect
+        async def mock_client_request(*args, **kwargs):
```

Ruff (0.13.1)

Unused function argument: `args`

(ARG001)